ui/src/components/UserList.tsx

```diff
@@ -23,7 +23,7 @@ const UserList: React.FC<Props> = ({users, onFollow}) => {
               <Icon
                 name='add user'
                 link
-                className='test-select-follow-icon'
+                className='test-select-add-user-icon'
```

Comment:
Isn't `add-user` to generic? What about
```suggestion
                className='test-select-follow-user-icon'
```

Reply:
I just named this based on the face that the icon is literally called an `'add user'` icon by Semantic UI. Also to have consistency with the icons next to the users' following and avoid `test-select-follow-user-following` - that is accurate on the other hand..

Reply:
@rohanjr This action leads to a user following another user, right? Then @hurryabit 's suggestion is OK

Reply:
Yeah it's fine. I could change to `'test-select-follow-user-icon'` next to your followers and `'test-select-follow-user-following-icon'` next to their following if you both prefer?

Reply:
Remember these are just for testing so I don't think it's a big deal, but I like nice names too